VideoLocker/src/main/java/org/edx/mobile/view/UserProfileFragment.java

```diff
@@ -114,11 +114,12 @@ protected void onSuccess(Account account) throws Exception {
 
                 @Override
                 protected void onException(Exception e) throws RuntimeException {
+                    viewHolder.loadingIndicator.setVisibility(View.GONE);
                     logger.error(e);
                     showErrorMessage(e);
                 }
             };
-            getAccountTask.setProgressDialog(viewHolder.loadingIndicator); // So that our indicator is hidden after task completes
```

Comment:
I guess not having it automatically handled is better suited for different solutions.